docs/output.md

```diff
@@ -142,9 +142,9 @@ For all samples:
 **Output directory: `results/Preprocessing/TSV`**
 
 - `duplicates_marked_no_table.tsv`, `duplicates_marked.tsv` and `recalibrated.tsv`
-    - `TSV` files to start `Sarek` from `prepare_recalibration`, `recalibrate` or `variantcalling` steps.
+    - `TSV` files to start `Sarek` from `prepare_recalibration`, `recalibrate` or `variant_calling` steps.
```

Comment:
```suggestion
    - `TSV` files to start `Sarek` from `preparerecalibration`, `recalibrate` or `variantcalling` steps.
```

Steps are renamed in the new version

Reply:
new version of what? The sarek.nf workflow uses variantcalling, but no longer includes the '_' handling/concatenation.